src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+
+Another negative side effect of large page weight is increased [latency](https://developer.mozilla.org/en-US/docs/Web/Performance/Understanding_latency) (the time it takes for the request to connect to the server storing the files and begin the process of transporting those files). If bunches of files are requested, no matter the technology, there is a limit on how much can be processed and transferred in any given period. I've audited WordPress sites that request as many as 170 files or more, which ensures terrible page loading performance starting with high latency periods.
+
+There are certainly optimizations that can be done, such as compressing and combining certain file requests, using HTTP/2 or the newer HTTP/3 protocols, and using a modern browser's ability to preconnect to and preload certain files to speed the the whole process process up. The [Performance chapter](https://almanac.httparchive.org/en/2021/performance) covers a wide range of factors that effect page loading performance.
+
+## Assets
+As explained in last years chapter, there's been little change in most assets in terms of technology, but there are some notable exceptions.
+
+### Static
+Static files reside by themselves and are used as resources to help build out and render web pages like video and audio files, and font files are all examples of static assets. [Images](https://almanac.httparchive.org/en/2021/media), which make a large percentage of the average web page's weight, are also a type of static asset file. So, let's use images for our example.
+
+Image file formats like PNG and JPEG are supported by browser vendors for rendering in web pages. More recent image formats, such as WebP, offer high quality, excellent compression (much better than PNG or JPG), and are supported by all major browsers have gained popularity. Some image types that are very good for the web lack full cross browser support so researching what image format will work with what browser is essential. Moreover, the latest browser features can look for the best choice for the browser requesting that image and defaulting back to a fully supported image if one is not available. The main point about images is to make sure they are optimized for the web, the [Media](https://almanac.httparchive.org/en/2021/media) chapter covers this in much more detail. Failing to properly size and compress images for your site will exact a high price on performance.
+
+**Note**: If you need an online service that will optimize your images, there is no better source I've found than Google's [Squoosh](https://squoosh.app/) application, and Jake Archibald's [SVGOMG](https://jakearchibald.github.io/svgomg/) for optimizing SVG's.
+
+#### A word about the proliferation in the use of JavaScript
```

Comment:
Just noting here that I'm going to stop doing "edits" on sections where I get where you're going with the content. For those that I have more "review" stuff, I'll still do some editorial suggestions.

Is this note supposed to be under the "static" subheading? Just double checking because the following section is confusing.